chain/icon/sender.go

```diff
@@ -64,14 +66,32 @@ type sender struct {
 }
 
 func (s *sender) newTransactionParam(prev string, rm *RelayMessage) (*TransactionParam, error) {
-	b, err := codec.RLP.MarshalToBytes(rm)
+	for i, bur := range rm.BlockUpdates {
+		s.l.Debugf("msg.BlockUpdates[%d]: %x\n", i, bur)
+	}
+
+	prm := &struct {
```

Comment:
The RelayMessage struct should be re-defined  ?
I don't see any point why do we need to create an anonymous struct here.